### PR TITLE
ON-34611 # Added formsAppEnvironmentId to NewTask

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,48 +1,24 @@
...
-  AWSTypes,
-  ApprovalTypes,
-  ConditionTypes,
-  PDFTypes,
-  SchedulingTypes,
-  CDNTypes,
-  EmailTemplateTypes,
-  FreshdeskTypes,
-  PDFConversionTypes,
-}
```
Comment (Zaxist) [inline index.d.ts:48]: Isn't this going to break everything that uses `FormTypes.Form` for example?

Comment (Zaxist) [reply index.d.ts:48]: Because `FormTypes` won't exist anymore, `Form` will just be exported from types

Comment (mymattcarroll) [reply index.d.ts:48]: `FormTypes` is being exported on line 2

Comment (Zaxist) [reply index.d.ts:48]: Oh yeh, I misread is as exporting all from.

Comment (Zaxist) [reply index.d.ts:48]: not as.

**Pull request merged**